main.go

```diff
@@ -129,6 +129,7 @@ func main() {
 			RayDashboardOAuthEnabled: ptr.To(true),
 			IngressDomain:            "",
 			MTLSEnabled:              ptr.To(true),
+			CertGeneratorImage:       "quay.io/project-codeflare/ray:latest-py39-cu118",
```

Comment:
I would like to change this to UBI9 pending validating that the init container successfully executes

Reply:
I have updated this to `registry.access.redhat.com/ubi9@sha256:770cf07083e1c85ae69c25181a205b7cdef63c11b794c89b3b487d4670b4c328` and the raycluster pods come up as expected with init containers running successfully with this image. 
I don't think it's related as I am getting the same error when using the other default image but when running the basic_interactive demo notebook I'm seeing `RuntimeError: Python minor versions differ between client and server: client is 3.9.16, server is 3.8.13` when runninng the cell containing ray.init.

Reply:
Also the e2e is failing, looking into it.

Reply:
When running it manually on openshift I'm seeing this error in the head pod `Permission denied: '/proc/driver/nvidia/gpus'`

Reply:
Needed to set mtls to false in e2e config. Working successfully with UBI9 image.